build.gradle

```diff
@@ -2,6 +2,8 @@ plugins {
 	id 'java'
 	id 'org.springframework.boot' version '3.0.6'
 	id 'io.spring.dependency-management' version '1.1.0'
+	id 'pmd'
+	id("org.owasp.dependencycheck") version "8.2.1"
```

Comment:
wolałbym, aby trzymać się jednego standardu.

Skoro powyżej jest np:
```
id 'io.spring.dependency-management' version '1.1.0'
```

- bez nawiasów ()
- pojedyncze apostrofy `'` zamiast `"`

to warto się tego schematu trzymać.